pkg/aks/create.go

```diff
@@ -97,7 +106,7 @@ func createManagedCluster(ctx context.Context, cred *Credentials, workplacesClie
 		logrus.Warnf("loadBalancerSKU 'basic' is not supported")
 		networkProfile.LoadBalancerSku = containerservice.Basic
 	case "":
-		networkProfile.LoadBalancerSku = containerservice.Standard
+		networkProfile.LoadBalancerSku = ""
```

Comment:
No major comment, but curious why we need to remove this?

Reply:
@furkatgofurov7 ,
Operator creates a standard loadbalancer (egress) by default, by removing this, I will not have egress created by uses pre-defined route table for egress.

Reply:
I am not sure about that, but I am going to test it